Apps/WebClient/src/ClientApp/src/models/ticket.ts

```diff
@@ -3,11 +3,11 @@ import { TicketStatus } from "@/constants/ticketStatus";
 export interface Ticket {
     id: string;
     room: string;
-    nonce?: string;
+    nonce: string;
     createdTime: number;
     checkInAfter: number;
     tokenExpires: number;
     queuePosition: number;
     status: TicketStatus;
-    token?: string;
+    token: string;
```

Comment:
I believe token can be null when status is Queued.

<img width="1446" alt="Screenshot 2022-12-15 at 4 15 16 PM" src="https://user-images.githubusercontent.com/58790456/207993309-037eae10-35f1-44a9-9acc-2d1357f983b0.png">

Reply:
Will correct.